esp/src/package.json

```diff
@@ -38,7 +38,9 @@
     "@hpcc-js/html": "2.8.2",
     "@hpcc-js/map": "2.14.7",
     "@hpcc-js/other": "2.13.12",
-    "@hpcc-js/tree": "2.9.1",
+    "@hpcc-js/tree": "2.11.0",
+    "@hpcc-js/codemirror": "2.12.3",
+    "@hpcc-js/phosphor": "2.13.15",
```

Comment:
@jbrundage When bumping the versions for @hpcc-js - its probably better to bump all of them to the latest,